gemfiles/mysql2/6-1.gemfile

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "rake"
+
+group :test do
+  platforms :jruby do
+    gem "activerecord-jdbcmysql-adapter", ">= 60.0.rc1"
+  end
+
+  platforms :ruby, :mswin, :mingw do
+    gem "mysql2"
+  end
+
+  gem "coveralls", require: false
+  gem "rspec", ">= 2.11"
+  gem "simplecov", require: false
+
+  gem "activerecord", "~> 6.1.0"
```

Comment:
This does not resolve until 6.1.0 is out. This does not match release candidates.

Reply:
Yes, you are correct.  I perhaps was jumping the gun with the PR, anticipating Rails 6.1 in the next few days and hoping to be able to upgrade when it arrives.